server/package.json

```diff
@@ -21,6 +21,7 @@
     "pg": "^8.12.0",
     "pg-hstore": "^2.3.4",
     "sequelize": "^6.37.3",
+    "server": "file:",
```

Comment:
???

Reply:
This dependency (server: "file") was automatically added by my editor. I didn’t manually make any changes, but it seems to be due to some settings or auto-completion in my IDE. If we don’t need it, we can safely remove it.

Reply:
if i take this branch and run `npm install`, that command fails.